use super::errors::AdventError;

pub fn solve(input: &str) -> Result<(), AdventError> {
    Ok(())
}